examples/access-control-migration/src/observers/sample.observer.ts

```diff
@@ -0,0 +1,92 @@
+// Copyright IBM Corp. 2020. All Rights Reserved.
+// Node module: @loopback/example-access-control-migration
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {
+  Application,
+  CoreBindings,
+  inject,
+  lifeCycleObserver, // The decorator
+  LifeCycleObserver,
+} from '@loopback/core';
+import * as _ from 'lodash';
+import {ProjectRepository} from '../repositories/project.repository';
+import {TeamRepository} from '../repositories/team.repository';
+import {UserRepository} from '../repositories/user.repository';
+
+/**
+ * This class will be bound to the application as a `LifeCycleObserver` during
+ * `boot`
+ */
+@lifeCycleObserver('')
+export class SampleObserver implements LifeCycleObserver {
+  constructor(
+    @inject(CoreBindings.APPLICATION_INSTANCE) private app: Application,
+    @inject('repositories.ProjectRepository')
+    private projectRepo: ProjectRepository,
+    @inject('repositories.TeamRepository') private teamRepo: TeamRepository,
+    @inject('repositories.UserRepository') private userRepo: UserRepository,
+  ) {}
+
+  /**
+   * This method will be invoked when the application starts
+   */
+  async start(): Promise<void> {
+    // Add your logic for start
+    if (process.env.SEED_DATA) {
+      await this.createUsers();
+      await this.createProjects();
+      await this.createTeams();
+    }
+  }
+
+  /**
+   * This method will be invoked when the application stops
+   */
+  async stop(): Promise<void> {
+    // Add your logic for stop
+  }
+
+  async createUsers(): Promise<void> {
+    const users = [
+      {id: 1, username: 'John', email: 'john@doe.com', password: 'opensesame'},
+      {id: 2, username: 'Jane', email: 'jane@doe.com', password: 'opensesame'},
+      {
+        id: 3,
+        username: 'Bob',
+        email: 'bob@projects.com',
+        password: 'opensesame',
+      },
+    ];
+
+    for (const u of users) {
+      await this.userRepo.create(_.pick(u, ['id', 'email', 'username']));
```

Comment:
The data is pre-generated in `db.json` by executing file, then this observer file is set to skip when start the app. I keep it because the original app has data creation script. 
see the explanation in doc https://github.com/strongloop/loopback-next/pull/4706/files#r383975344